### PR TITLE
ex5_ch2.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Instructions: Write a program that prompts for two numbers. Print the sum, difference, product, and quotient of those numbers as shown in the example output:

Example Output:
       What is the first number? 10
       What is the second number? 5
       10 + 5 = 15
       10 - 5 = 5
       10 * 5 = 50
       10 / 5 = 2

### CODE REVIEW

File: ex5_ch2.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Created on Wed Jun 13 10:08:53 2018
+
+@author: amyhaddad
+"""
+
+import math 
+
+#step1: write input statement asking for one number
+first_number = int(input("This is your first number: "))
+
```
Comment (amymhaddad) [inline ex5_ch2.py]: Question: Based on my question, I'm assuming the user inputs a number. However, what if they write out a number, like "seven." How do I convert the string to an integer if the answer will always be different each time?

Comment (paulghaddad) [reply ex5_ch2.py]: You have to make the assumption that they will type it in correctly. Down the road, you'll learn ways to catch bad input and tell the user to enter it in differently. But for now, I'd just focus on getting the program to work properly with good input. 

So this line is exactly how I'd do it!

File: ex5_ch2.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#step2: write input statement asking for second number
+second_number = int(input("This is your second number: "))
+
+#step2: create variable "sum" and write an equation that adds the two numbers together and print equation and solution 
+sum = first_number + second_number 
+
+print(f"{first_number} + {second_number} = {sum}")
```
Comment (paulghaddad) [inline ex5_ch2.py]: This is excellent! Nice work!

File: ex5_ch2.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import math 
+
+#step1: write input statement asking for one number
+first_number = int(input("This is your first number: "))
+
+#step2: write input statement asking for second number
+second_number = int(input("This is your second number: "))
+
+#step2: create variable "sum" and write an equation that adds the two numbers together and print equation and solution
```
Comment (paulghaddad) [inline ex5_ch2.py]: Nice comments. Right now, it's good to think about breaking your solutions into small chunks, as you're doing.

**Pull request merged**